src/map/entities/battleentity.h

```diff
@@ -423,6 +423,9 @@ enum IMMUNITY : uint16
     IMMUNITY_REQUIEM     = 0x400,  // 1024
     IMMUNITY_LIGHT_SLEEP = 0x800,  // 2048
     IMMUNITY_DARK_SLEEP  = 0x1000, // 4096
+    IMMUNITY_ASPIR = 1 << 13,     // 8192
```

Comment:
Should follow the previously used styling for this.  While I'm a fan of declarations using bitshifts, it doesn't follow current formatting